sysbuild/Kconfig.mcuboot

```diff
@@ -98,7 +98,8 @@ config MCUBOOT_BUILD_DIRECT_XIP_VARIANT
 
 menuconfig MCUBOOT_HARDWARE_DOWNGRADE_PREVENTION
 	bool "Downgrade prevention using hardware security counters"
-	depends on (BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS || BOARD_THINGY53_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP_NS || BOARD_NRF9160DK_NRF9160 || BOARD_NRF9160DK_NRF9160_NS || BOARD_NRF9161DK_NRF9161 || BOARD_NRF9161DK_NRF9161_NS || BOARD_THINGY91_NRF9160 || BOARD_THINGY91_NRF9160_NS || BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF7002DK_NRF5340_CPUAPP_NS)
+	depends on (SOC_NRF5340_CPUAPP || SOC_SERIES_NRF91X)
+	depends on !SECURE_BOOT_APPCORE
```

Comment:
why?

Reply:
they both use the KMU data section, so they both conflict?